Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -733,6 +734,10 @@ class BrowserViewController: UIViewController,
         var fakespotNeedsUpdate = false
         if urlBar.currentURL != nil {
             fakespotNeedsUpdate = contentStackView.isSidebarVisible != FakespotUtils().shouldDisplayInSidebar(viewSize: size)
+            if isReduxIntegrationEnabled,
+                let fakespotState = store.state.screenState(FakespotState.self, for: .fakespot) {
```

Comment:
Keeping a version of the most recent state around will avoid accessing it like this which I'm not sure it's intended.